Update Pay and Invoice redirect pages

Christinadobrzyn opened the pull request:
Redirecting https://help.expensify.com/articles/new-expensify/expenses-&-payments/pay-an-invoice.html to https://help.expensify.com/articles/new-expensify/expenses-&-payments/Pay-an-invoice


### Details
We need to redirect https://help.expensify.com/articles/new-expensify/expenses-&-payments/pay-an-invoice.html to https://help.expensify.com/articles/new-expensify/expenses-&-payments/Pay-an-invoice. 

Added the redirect based on this [SO](https://stackoverflowteams.com/c/expensify/questions/18514/18515#18515). 

### PR Author Checklist

- [X] I linked the correct issue in the `### Fixed Issues` section above
- [X] I wrote clear testing steps that cover the changes made in this PR
    - [X] I added steps for local testing in the `Tests` section
    - [X] I added steps for the expected offline behavior in the `Offline steps` section
    - [X] I added steps for Staging and/or Production testing in the `QA steps` section
    - [X] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [X] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [X] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [X] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [X] I ran the tests on **all platforms** & verified they passed on:
    - [X] Android: Native
    - [X] Android: mWeb Chrome
    - [X] iOS: Native
    - [X] iOS: mWeb Safari
    - [X] MacOS: Chrome / Safari
    - [X] MacOS: Desktop
- [X] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [X] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [X] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [X] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [X] I verified that comments were added to code that is not self explanatory
    - [X] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [X] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [X] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [X] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [X] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [X] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [X] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [X] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [X] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [X] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [X] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [X] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [X] I verified that if a function's arguments changed that all usages have also been updated correctly
- [X] If any new file was added I verified that:
    - [X] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [X] If a new CSS style is added I verified that:
    - [X] A similar style doesn't already exist
    - [X] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG)`)
- [X] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [X] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [X] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [X] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [X] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [X] I verified that all the inputs inside a form are aligned with each other.
    - [X] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [X] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [X] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.

Christinadobrzyn committed:
Update redirects.csv

Redirecting https://help.expensify.com/articles/new-expensify/expenses-&-payments/pay-an-invoice.html to https://help.expensify.com/articles/new-expensify/expenses-&-payments/Pay-an-invoice

Christinadobrzyn committed:
Update Pay-an-invoice.md

Making the FAQ section collapsable. Removing FAQ and adding {% include faq-begin.md %}

Christinadobrzyn committed:
Update Approve-and-pay-expenses.md

Trying to make the bullet points at the top of the page so they are stacked on top of each other instead of on one line

rushatgabhane commented on docs/articles/new-expensify/expenses-&-payments/Approve-and-pay-expenses.md:
```diff
@@ -6,8 +6,8 @@ description: Approve, hold, or pay expenses submitted to you
-- Instantly submitted without needing approval.
-- Submitted at a desired frequency (daily, weekly, monthly) and follow an approval workflow. 
+- Instantly submitted without needing approval.
+- Submitted at a desired frequency (daily, weekly, monthly) and follow an approval workflow. 
```
```suggestion
- Submitted at a desired frequency (daily, weekly, monthly) and follow an approval workflow. 
```

rushatgabhane replied:
```suggestion
- Submitted at a desired frequency (daily, weekly, monthly) and follow an approval workflow. 
```

rushatgabhane commented on docs/articles/new-expensify/expenses-&-payments/Approve-and-pay-expenses.md:
```diff
@@ -6,8 +6,8 @@ description: Approve, hold, or pay expenses submitted to you
 
-- Instantly submitted without needing approval.
-- Submitted at a desired frequency (daily, weekly, monthly) and follow an approval workflow. 
+- Instantly submitted without needing approval.
```
```suggestion
- Instantly submitted without needing approval.
```

rushatgabhane replied:
```suggestion
- Instantly submitted without needing approval.
```

Christinadobrzyn committed:
Update Approve-and-pay-expenses.md

Trying to update the bullets at the top of the page to 

- Submitted at a desired frequency (daily, weekly, monthly) and follow an approval workflow.

rushatgabhane commented:
close the PR please. changes done here - https://github.com/Expensify/App/pull/49011/files

dangrous commented:
Closing as per @rushatgabhane's comment - @Christinadobrzyn if you need anything else let me know!
